specification/cognitiveservices/data-plane/Language/preview/2022-02-01-preview/ConversationalAnalysis-authoring.json

```diff
@@ -0,0 +1,2742 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Language Authoring CLU APIs",
+    "version": "2022-02-01-preview"
+  },
+  "x-ms-parameterized-host": {
+    "hostTemplate": "{Endpoint}/language/analyze-conversations",
+    "useSchemePrefix": false,
+    "parameters": [
+      {
+        "$ref": "#/parameters/Endpoint"
+      }
+    ]
+  },
+  "securityDefinitions": {
+    "apiKeyHeader": {
+      "type": "apiKey",
+      "name": "Ocp-Apim-Subscription-Key",
+      "in": "header"
+    }
+  },
+  "security": [
+    {
+      "apiKeyHeader": []
+    }
+  ],
+  "paths": {
+    "/projects": {
+      "get": {
+        "description": "Returns the list of projects.",
+        "operationId": "Projects_GetProjectsList",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "query",
+            "name": "top",
+            "description": "The number of projects to be returned.",
+            "type": "string"
+          },
+          {
+            "in": "query",
+            "name": "skip",
+            "description": "The offset in the response.",
+            "type": "string"
+          },
+          {
+            "in": "query",
+            "name": "maxpagesize",
+            "description": "The maximum number of projects per page.",
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The metadata of projects.",
+            "schema": {
+              "$ref": "#/definitions/ProjectGetObjectList"
+            }
+          },
+          "400": {
+            "description": "This error can be returned if the request's parameters are incorrect meaning the required parameters are missing, malformed, or too large.\\r\\n\\r\\nThis error can be returned if the request's body is incorrect meaning the JSON is missing, malformed, or too large."
+          },
+          "401": {
+            "description": "You do not have access. \\r\\n\\r\\nReasons can include:\\r\\n\\r\\n* used endpoint subscription key, instead of authoring key\\r\\n* invalid, malformed, or empty authoring key\\r\\n* authoring key doesn't match region\\r\\n* you are not the owner or collaborator\\r\\n* invalid order of API calls."
+          },
+          "403": {
+            "description": "Total monthly key quota limit exceeded."
+          },
+          "429": {
+            "description": "Rate limit is exceeded."
+          }
+        }
+      }
+    },
+    "/projects/{projectName}": {
+      "patch": {
+        "description": "Creates a new or update a project.",
+        "operationId": "Projects_CreateOrUpdateProject",
+        "consumes": [
+          "application/json"
```

Comment:
JSON Merge Patch (RFC7396) is recommended. https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#json-resource-schema--field-mutability